x-pack/plugins/monitoring/server/plugin.ts

```diff
@@ -389,18 +393,23 @@ export class MonitoringPlugin
           }
         };
 
-        const validate: any = get(options, 'config.validate', false);
-        if (validate && validate.payload) {
-          validate.body = validate.payload;
-        }
+        const validate: MonitoringRouteConfig<Params, Query, Body, Method>['validate'] =
+          // NOTE / TODO: "config.validate" is a legacy convention and should be converted over during the TS conversion work
```

Comment:
Might be good to reference a github issue here to help provide more context on what's next for this TODO.

Reply:
Yeah, I've altered the approach and removed the "workarounds" on the types for `config.validate` and `method`. There isn't actually any need to have them since the offending files would be `.js` anyway. This will help others picking up the other conversion issues since the compiler will just shout as soon as the file extension is changed.

It means that after all of the conversions there will be two things to cleanup in `plugin.ts`:

```
const validate: MonitoringRouteConfig<Params, Query, Body, Method>['validate'] =
          // NOTE / TODO: "config.validate" is a legacy convention and should be converted over during the TS conversion work
          get(options, 'validate', false) || get(options, 'config.validate', false);
```

This will just become one check.

And `method.toLowerCase()` won't be needed as all the methods will adhere to the new core Kibana type.